src/mscorlib/facade/mscorlib.csproj

```diff
@@ -97,6 +88,42 @@
     <SeedTypePreference Include="System.Console">
       <Assembly>System.Console</Assembly>
     </SeedTypePreference>
+    <SeedTypePreference Include="System.IO.BinaryReader">
```

Comment:
I thought I looked at an earlier version of this PR where you removed these from CoreLib. What triggered you to add them back?